### DISCUSSION TITLE
Using CSLA 5: Blazor and WebAssembly book revision #

### DISCUSSION THREAD

**Discussion by ghost** [MarimerLLC/csla#1774] [Questions]
I currently have Revision: 200519 of the book. Is this the most current version?

Thanks

**Comment by rockfordlhotka** [comment]
Yes, that is the current/final version of the book.

**Comment by ghost** [comment]
Great, thanks!

On page 103 (Chapter 6: Multi-Headed Blazor Solutions), it mentions adding a reference to "Microsoft.AspNetCore.Blazor.HttpClient" for the shared project, but it doesn't look that's a valid package anymore. What would be the correct package to add?

**Reply by rockfordlhotka** [reply]
The [`BlazorMultiHead.Ui` project ](https://github.com/MarimerLLC/BlazorBook/blob/master/BlazorMultiHead/BlazorMultiHead.Ui/BlazorMultiHead.Ui.csproj) contains that package reference to the prerelease version from the time.

If they still haven't released the package, it would only show up in Visual Studio if you select the option to display prerelease packages. Maybe that is the problem?

**Comment by ghost** [comment]
Yes, it's prerelease. But Blazor WASM isn't supposed to be prerelease anymore. That's why I was wondering if that was still the right package to use.
